gef-ui/server/controllers/about-facility/index.test.js

```diff
@@ -786,7 +837,11 @@ describe('controllers/about-facility', () => {
       mockRequest.body['cover-end-date-month'] = format(threeMonthsAndOneDayFromNow, 'M');
       mockRequest.body['cover-end-date-year'] = format(threeMonthsAndOneDayFromNow, 'yyyy');
 
-      await validateAboutFacility(mockRequest, mockResponse);
+      if (isFacilityEndDateEnabledOnDeal(getDefaultGefDealVersion())) {
```

Comment:
considering these are unit tests, I wonder if we should run a v0 and a v1, but lets catch up on that as it may be a bad idea that ive not thought fully about

Reply:
Probably a good idea - would make this a 1500+ line file (:

Reply:
Would add only a few lines with a `forEach` and adding a custom function to mock the response of `getDefaultGefDealVersion`?

Reply:
Or we could split it into baseTests, versionSpecific tests...ie withEveryDealVersionTests, withVersion0Tests, withVersion1Tests? Maybe, or maybe theres a nicer way?